homeassistant/components/dremel_3d_printer/strings.json

```
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        },
+        "title": "Connect with your Dremel 3D Printer"
```

Doesn’t add much value.
```suggestion
        }
```